Resources/config/doctrine/BasePage.orm.xml

```diff
@@ -22,6 +22,7 @@
         <field name="templateCode" type="string" column="template" nullable="false"/>
         <field name="createdAt" type="datetime" column="created_at"/>
         <field name="updatedAt" type="datetime" column="updated_at"/>
+        <field name="targetRedirectType" type="integer" column="target_redirect_type" nullable="true" default="302"/>
```

Comment:
You introduce a new field, which means people have to generate and run migrations. Will the software continue to work if they forget to do that? If not, this is a BC-break.